### PR TITLE
V5

### PR DESCRIPTION
**Pull request by jonwinton**
# API Changes
- Underscores routes
- Renderers now terminate the Express response
- Removed references to `resolveMedia`
- Removed the check for memory leaks from `control`
- Deprecate the old publish API

# Other Changes
- Circle tests on 8.9.4
- Removed the `winston` logger completely from tests

### CODE REVIEW

File: lib/render.test.js [hunk 1]
```diff
@@ -183,18 +148,18 @@ describe(_.startCase(filename), function () {
...
-      db.get.returns(resolveString({layout: '/components/hey'}));
+      db.get.returns(resolveString({layout: '/_components/hey'}));
 
       // 2. get the data that will be composed in the template
       components.get.returns(bluebird.resolve({}));
 
-      // 3. get the template that will be composed
-      components.getTemplate.returns('asdf');
+      // // 3. get the template that will be composed
+      // components.getTemplate.returns('asdf');
```
Comment (byronhulcher) [inline lib/render.test.js]: should this (and other commented out lines from this PR) be removed from the test file? Hard to tell if a commented out part of a test is temporarily broken or purposeful removal

Comment (jonwinton) [reply lib/render.test.js]: Oh, good find. Forgot that was there.

File: lib/services/publish.test.js [hunk 2]
```diff
@@ -7,11 +7,10 @@ const _ = require('lodash'),
   db = require('../services/db'),
   lib = require(`./${filename}`);
 
-require('sinon-as-promised');
+// require('sinon-as-promised');
```
Comment (byronhulcher) [inline lib/services/publish.test.js]: can be removed?

Comment (jonwinton) [reply lib/services/publish.test.js]: Yep! Will also remove. So much deprecation. This is a reminder to myself to also check that I removed `winston`

**Pull request merged**